docs/prow/prow-jobs-quick-start.md

```diff
@@ -0,0 +1,69 @@
+# Prow Jobs QuickStart
+
+This document provides an overview of how to quickly start working with Prow jobs.
+
+1. Fork [`test-infra`](https://github.com/kyma-project/test-infra) repository and feature a new branch.
+
+
+2. Jobs are generated from templates. To create one, add `<PROW JOB NAME>-data.yaml` file in the `templates/data` directory. The file should look like this:
+
+    ```yaml
+    templates:
+      - from: templates/generic.tmpl
+        render:
+          - to: ../prow/jobs/test-infra/stability-checker.yaml
+        <CONFIGURATION>
+    ```
+    In the `<CONFIGURATION>` part you can specify local config sets (**localSets**) and a configuration of a single job (**jobConfig**), where e.g., the name of the job can be defined.
+    If needed, global config sets (**globalSets**) can be added to the `templates/config.yaml` file.
+    
+    > **NOTE:** Make sure that your template file and Prow job have unique names.
+    
+    - To learn more about **localSets**, **jobConfig** and **globalSets**, please refer to [specific documentation](https://github.com/kyma-project/test-infra/tree/main/development/tools/cmd/rendertemplates). 
+    - You can search for more examples of template files in the `templates/data` directory.
+
+
+3. Render template with one of these commands:
+    ```bash
+    go run development/tools/cmd/rendertemplates/main.go --data path/to/directory/with/data/files
+    ```
+    or 
+    ```bash
+    make jobs-definitions
+    ```
+    
+    - For more details on how rendering templates works, see [this](https://github.com/kyma-project/test-infra/tree/main/development/tools/cmd/rendertemplates) document.
+    
+    > **NOTE:** Do not change generated file!
```

Comment:
```suggestion
    > **CAUTION:** Do not change the generated file!
```
What would happen if the user changes it? Maybe we should mention the consequences.

Reply:
PR wouldn't be merged, because the job checking the generated file would fail.